packages/backend/src/accounting/psql/service.ts

```diff
@@ -38,7 +38,7 @@ import { TelemetryService } from '../../telemetry/service'
 
 export interface ServiceDependencies extends BaseService {
   knex: TransactionOrKnex
-  telemetry: TelemetryService
+  telemetry?: TelemetryService
```

Comment:
This should not be optional. I think maybe it ended up this way after resolving a merge conflict or something?

[It used to be optional but we refactored it to always be required (but do nothing if disabled) in this PR](https://github.com/interledger/rafiki/pull/2991).

After making it required we can also change all the timer calls like:

```ts
const stopTimer = deps.telemetry?.startTimer(...)
...
stopTimer && stopTimer()
```

to

```ts
const stopTimer = deps.telemetry.startTimer(...)
...
stopTimer()
```